control.lua

```diff
@@ -163,7 +166,15 @@ local function onAirFiltering()
             if (entity.is_connected_to_electric_network() and
                 ((entity.energy / entity.prototype.max_energy_usage) > 0.65)) then
                 local amount = entity.surface.get_pollution(entity.position)
-                if (amount > 0) then
+                if not minDiffuse then
+                    minDiffuse = game.map_settings.pollution.min_to_diffuse
+                end
```

Comment:
Why have this check in the logic, wont the value always be set before this logic is run?

Reply:
It will be set only at game start. I didn't find how to get it at game load (forbidden to do at on_load). Correct me please if needed according with your previous comment.